libs/GraphicECS/SFML/Components/SoundName.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** SoundName
+*/
+
+#ifndef SOUNDNAME_HPP_
+#define SOUNDNAME_HPP_
+
+#include "Component/Component.hpp"
+#include "GraphicECS/SFML/Resources/SoundResource.hpp"
+
+namespace ecs
```

Comment:
Bad namespace, use graphicECS::SFML::Components

Reply:
improve in #205 and #206 but yes i change that this afternoon